lib/bolt/cli.rb

```diff
@@ -279,6 +279,11 @@ def execute(options)
         return 0
       end
 
```

Comment:
We should not load  inventory for plan convert. We should add this logic to https://github.com/puppetlabs/bolt/blob/d1aed20d51b3d6a85a312dd2c33cebe76adcfe18/lib/bolt/cli.rb#L122-L124 (maybe something a bit cleaner than just adding another `or` condition). 

Similarly we could be sneaky for now (before we start collecting analytics on this) and avoid loading inventory for analytics (and building analytics client all together) my moving the logic defined in line 282-286 to https://github.com/puppetlabs/bolt/blob/10e9bfc5a8fa8bb315418e99fbb70934f901c126/lib/bolt/cli.rb#L246

Reply:
This should be part of the acceptance criteria in the future.